routes.go

```diff
@@ -29,3 +34,23 @@ func setupRoutes(pubsubClient *pubsub.Client) {
 func routeNotFoundHandler(w http.ResponseWriter, r *http.Request) {
 	service.RespondWithNotFoundError(w, "Not Found")
 }
+
+func dropTraceMiddleware() func(http.Handler) http.Handler {
+	header := os.Getenv("DROP_TRACE_HEADER")
+	if header == "" {
+		return func(next http.Handler) http.Handler { return next }
+	}
+
+	return func(next http.Handler) http.Handler {
+		return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			if r.Header.Get(header) != "" {
+				span, _ := tracer.StartSpanFromContext(r.Context(), "Drop trace")
```

Comment:
Can't we just grab the current span instead of starting one? 🤔

Reply:
If I remember correctly from where we did this first, it is safer not to assume there's a span available already (also, I don't remember if there's an API for retrieving the current one with the context we have available).